client/src/main/scala/org/apache/celeborn/client/ChangePartitionManager.scala

```diff
@@ -165,6 +168,7 @@ class ChangePartitionManager(
     // check if there exists request for the partition, if do just register
     val requests = changePartitionRequests.computeIfAbsent(shuffleId, rpcContextRegisterFunc)
     inBatchPartitions.computeIfAbsent(shuffleId, inBatchShuffleIdRegisterFunc)
+    shuffleIsSegmentGranularityVisible.put(shuffleId, isSegmentGranularityVisible)
```

Comment:
I believe the ChangePartitionManager can directly call lifecycleManager.commitManager.isSegmentGranularityVisible to avoid redundant metadata in memory. WDYT?

Reply:
> I believe the ChangePartitionManager can directly call lifecycleManager.commitManager.isSegmentGranularityVisible to avoid redundant metadata in memory. WDYT?

Thank you for your careful review. I have modified it based on your suggestions.